test/box-luatest/gh_8146_space_and_index_name_in_iproto_requests_test.lua

```diff
@@ -0,0 +1,86 @@
+local netbox = require('net.box')
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new({alias = 'dflt'})
```

Comment:
Nit: `alias` is optional - you may skip it, in which case it'll be set to "server".

Reply:
Thanks, I didn't know about this! Fixed.